platform/kube/inject/inject.go

```diff
@@ -460,6 +467,17 @@ func intoObject(c *Config, in interface{}) (interface{}, error) {
 		}
 	}
 
+	// skip customized exclude namespaces
+	for _, excludeNamespace := range c.ExcludeNamespaces {
+		if excludeNamespace == "" {
+			return out, nil
+		}
```

Comment:
why do you need this check? is it for the case where someone specifies
`excludeNamespaces: [""]` ? Because that is as good as not running the initializer. Might as well disallow that option right?

Reply:
Done

Reply:
This shouldn't be needed anymore with the check on line 208?